geoapi/tools/check_observable_projects.py

```diff
@@ -0,0 +1,26 @@
+from geoapi.models import ObservableDataProject, Project
+from geoapi.db import create_task_session
+from geoapi.log import logger
+
+# Check that migrations worked out as expected
+# Remove this file with https://tacc-main.atlassian.net/browse/WG-377
+
+with create_task_session() as session:
+    observable_projects = session.query(ObservableDataProject).all()
```

Comment:
This feels off after replacing the refs to `observable...` in other files. Don't really have a great alternative, but maybe something like `watched_projects` to be consistent with the language used in other areas?

Reply:
I think this one is okay. as we'll remove the script once we remove the ObservableDataProject table. I got a follow-on jira issue for it:  https://tacc-main.atlassian.net/browse/WG-377